Libraries/StyleSheet/PlatformColorValueTypes.macos.js

```diff
@@ -7,7 +7,9 @@
  * @format
  * @flow strict-local
  */
-// [TODO(macOS GH#774)
+
+// [macOS]
```

Comment:
Should this be moved up into the block comment at the top of the file

Reply:
Same thing in the two files below

Reply:
Never mind, read the md file you wrote incorrectly, this is right, going to resolve